templates/README.md.gotmpl

```diff
@@ -83,6 +83,21 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
+| [![CIS KUBERNETES]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/cis_kubernetes" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=CIS+KUBERNETES+V1.5" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Center for Internet Security, KUBERNETES Compliance |
+| [![CIS AWS]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/cis_aws" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=CIS+AWS+V1.2" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Center for Internet Security, AWS Compliance |
+| [![CIS AZURE]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/cis_azure" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=CIS+AZURE+V1.1" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Center for Internet Security, AZURE Compliance |
+| [![PCI-DSS]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/pci" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=PCI-DSS+V3.2" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Payment Card Industry Data Security Standards Compliance |
+| [![NIST-800-53]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/nist" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=NIST-800-53" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | National Institute of Standards and Technology Compliance |
+| [![ISO27001]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/iso" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=ISO27001" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Information Security Management System, ISO/IEC 27001 Compliance |
+| [![SOC2]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/soc2" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=SOC2" (ds "config").github_repo | regexp.Replace "/" "%2F"}})| Service Organization Control 2 Compliance |
+| [![CIS GCP]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/cis_gcp" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=CIS+GCP+V1.1" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Center for Internet Security, GCP Compliance |
+| [![HIPAA]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/hipaa" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=HIPAA" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Health Insurance Portability and Accountability Compliance |
```

Comment:
Replacing `/` with `%2f` is creating invalid URLs (E.g. `https:%2f%2fwww.bridgrcrew....`)

Reply:
fixed